Displayed current Track Album and Artist on left of screen

3Ustin opened the pull request:
We decided to get the outline of these UI components before getting in the greater detail. This PR is an example of that strategy.
This PR resolved Issue #27

3Ustin committed:
Displayed current Track Album and Artist on left of screen

We decided to get the outline of these UI components before getting in the greater detail. This PR is an example of that strategy.

3Ustin committed:
Changed updated for PR

We decided having Uniform Quotations across the platform would be best.

MtuMartian commented on src/components/PlayerUI.tsx:
```diff
@@ -13,11 +16,13 @@ function PlayerUI(props: IPlayerUIProps) {
+            <Toolbar sx={{ flexDirection:"column" }}>
+                <Container  sx={{padding:'20px', paddingTop:'40px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Name: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{trackName}</Typography></Container>
+                <Container sx={{padding:'20px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Album: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{albumName}</Typography></Container>
+                <Container sx={{padding:'20px'}}><Typography variant='h5' align='center' sx={{color:'rgb(0, 0, 0)'}}>Artist: </Typography><Typography align='center' sx={{color:'rgb(0, 0, 0)'}}>{artistName}</Typography></Container>
```
Instead of using inline styles with the `sx` attribute, can we use css classes? This will allow us to remove some of the clutter in our components and to reduce repeated code

3Ustin committed:
PR request changes

The AppBar component is rewriting the imported css code so the AppBar is inline for now.
